### ISSUE TITLE
Option to use data-lines (or similar) on the element 

### ISSUE THREAD

**Issue by m9dfukc** [rviscomi/trunk8#12]
Would be nice to define a data element on the tag which states the amount of lines it should shorten to. Doing this manually in a loop for now but would be nice to have this build in.

**Comment by woozyking** [comment]
:+1:

**Comment by fchristant** [comment]
I agree it would be a nice feature. For now, I'm also using this loop, which is simply enough:

$('[data-truncate="true"]').each(function( index, element ) {
		$(element).trunk8({
			lines: $(this).data("truncate-lines")
		});
	});
